conjureup/app.py

```diff
@@ -274,10 +276,29 @@ def main():
     with open(spells_index_path) as fp:
         app.spells_index = yaml.safe_load(fp.read())
 
+    addons_aliases_index_path = os.path.join(app.config['spells-dir'],
+                                             'addons-aliases.yaml')
+    with open(addons_aliases_index_path) as fp:
+        app.addons_aliases = yaml.safe_load(fp.read())
```

Comment:
This should probably handle ENOENT errors (or check for existence) on the off chance that someone uses this with a spells repo that doesn't have any aliases.